bridge/package.json

```diff
@@ -64,6 +64,7 @@
     "d3": "7.6.1",
     "d3-scale": "4.0.2",
     "d3-shape": "3.1.0",
+    "eslint-plugin-no-only-tests": "3.0.0",
```

Comment:
Probably my fault, but this dependency should be a dev dependency.
`yarn add -D eslint-plugin-no-only-tests@3.0.0`

Reply:
ah, nice catch! I've applied this in a force-push to keep the diff clean so far